src/templates/components/staffProfileSideBarNav/index.test.tsx

```diff
@@ -21,32 +20,57 @@ describe('SidebarNav', () => {
     },
   }
 
+  const mockLovellSidebarData = {
+    link: {
+      label: 'Donald Koenig',
+      url: { path: '/' },
+    },
+    parent: {
+      label: 'Leadership',
+      links: [],
+      url: { path: '/lovell-federal-health-care/leadership' },
+    },
+  }
   it('renders the sidebar navigation with the correct structure', () => {
     render(
-      <StaffProfileSideBarNav sidebarData={mockSidebarData} entityUrlPath="/" />
+      <StaffProfileSideBarNav
+        sidebarData={mockSidebarData}
+        lovellVariant={undefined}
+      />
     )
-
+    const leaderShipLink = screen.getByTestId('sidebar-nav-link')
```

Comment:
Ohhh, I see. Because this is finding the `<va-link>`. Using the `data-testid` is totally fine, but an alternative I've done is like this:
```suggestion
    const leaderShipLink = container.querySelector('va-link[text="Leadership"]')
```

You'd have to get `container` from `render()`, of course:
```ts
const { container } = render(...)
```

Going this route also checks that it's a `<va-link>` _and_ the text is set as we'd expect.